<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HashForm</class>
 <widget class="QWidget" name="HashForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabFileCheck">
      <attribute name="title">
       <string>文件校验</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelTotal">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>总进度：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBarTotal">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelFileProgress">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>文件进度：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBarFile">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="plainText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_1">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonOpen">
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>93</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCopy">
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>93</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>复制</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSaveAs">
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>93</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>另存为</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonClear">
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>93</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" rowspan="3">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="checkBoxMD5">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>MD5</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxSha1">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sha1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxSha256">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sha256</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxSha512">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sha512</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
